### ISSUE TITLE
Connecting to MQTT at mqtt://mqtt

### ISSUE THREAD

**Issue by rsissons** [stjohnjohnson/smartthings-mqtt-bridge#70]
[11:50:35] pi@roosterpi1:~$ smartthings-mqtt-bridge
info: Starting SmartThings MQTT Bridge - v1.3.4
info: Loading configuration
info: Loading previous state
info: Perfoming configuration migration
info: Saving current state
info: Connecting to MQTT at **mqtt://mqtt**

Shouldn't this be Connecting to MQTT at mqtt://localhost..? It's not connecting. Where can I update this to point to my local server?

Thanks!

**Comment by johnypony3** [comment]
RTFM, there is a configuration file that needs to be updated.

**Comment by rsissons** [comment]
I don't see in the readme where it says that. Thanks for being helpful.

**Comment by johnypony3** [comment]
Lets not dice things, you cant expect it to magically know where your queue is.
[Configuration](https://github.com/stjohnjohnson/smartthings-mqtt-bridge#configuration)

Please close.

**Comment by rsissons** [comment]
After trying to use the instruction you linked to, that I saw before, it doesn't work. All I changed was the host to localhost, IP address, and neither worked and the HA wouldn't even start up. 

So after researching, I found Bruh's MQTT setting that seem to work except it still wants to connect to MQTT:MQTT instead of my localhost.

Here is my configuration.yaml MQTT section. 
mqtt:
  broker: 192.168.1.206
  port: 1883
  client_id: home-assistant-1
  username: pi
  password: xxxxxxxxxx
  birth_message:
    topic: 'homeassistant/status'
    payload: 'online'
    qos: 1
    retain: true
  will_message:
    topic: 'homeassistant/status'
    payload: 'offline'
    qos: 1
    retain: true

HA seems to like this and I can toggle the switch back and forth and can see the messages being received it just doesn't know about connecting to the localhost.

The link you sent, is it suppose to exist as a separate yaml file? Am I not supposed to use the MQTT section in my main configuration.yaml?
